lib/reek/smells/smell_repository.rb

```diff
@@ -25,6 +25,7 @@ def self.smell_types
       # @return [Array<Reek::Smells::SmellDetector>] All SmellDetectors that we want to filter for
       #         e.g. [Reek::Smells::Attribute].
       def self.eligible_smell_types(filter_by_smells = [])
+        filter_by_smells = Array(filter_by_smells)
```

Comment:
I do this to ensure we have an array, so the `#include?` later on uses the Array version, rather than the String version. This stops mutant from suggesting removal of quite a bit of code elsewhere. However, this is a bit of a leftover from an earlier direction I took in implementing this, and it can go now.